opencog/nlp/chatbot/sentence-matching.scm

```diff
@@ -0,0 +1,32 @@
+; Returns one or more sentences that are structurally similar to the input one.
```

Comment:
Understood...

Man Hin decided it made sense to merge the code now and then later merge a
version with a better tuned algorithm.  I think this was a good choice...

On Sat, Feb 28, 2015 at 11:11 AM, Linas Vepštas <notifications@github.com>
wrote:

> In opencog/nlp/chatbot/sentence-matching.scm
> <https://github.com/opencog/opencog/pull/1324#discussion_r25553559>:
>
> > @@ -0,0 +1,32 @@
> > +; Returns one or more sentences that are structurally similar to the input one.
>
> Yeah, I didn't review and am not critiquing the algorithm. I only reviewed
> superficial details, e.g. does it fit? is it well organized? Cleanly coded?
> etc. Mostly its clean, it fits and is well organized. If the algorithm is
> horrible, we'll find out later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/1324/files#r25553559>.
>



-- 
Ben Goertzel, PhD
http://goertzel.org

"The reasonable man adapts himself to the world: the unreasonable one
persists in trying to adapt the world to himself. Therefore all progress
depends on the unreasonable man." -- George Bernard Shaw